### ISSUE TITLE
Accessibility: Visibility of chain symbol

### ISSUE THREAD

**Issue by saskiaw** [Redocly/redoc#2523]
**Describe the problem to be solved**
Headings with a 'chain symbol' serve as links and receive focus when tabbing. However, the focus indicator for the chain symbol is not visible. Therefore, when navigating with the keyboard (tab), you can access the links, and only then does the chain symbol become visible.

![image](https://github.com/Redocly/redoc/assets/74241382/df507ba7-e4b3-4959-a120-fb99861466ce)

**Describe the solution you'd like**
We would like the chain symbol to always be visible and to have a border around it when focused with the tab key.
